src/app/business/block/buckets.component.ts

```diff
@@ -208,8 +208,8 @@ export class BucketsComponent implements OnInit{
                                 label:item.name,
                                 value:item.name
                             });
-                            item.encryptionEnabled = item.SSEConfiguration.SSE.enabled.toLower() == "true" ? true : false;
-                            item.versionEnabled = item.VersioningConfiguration.Status.toLower() == "enabled" ? true : false;
+                            // item.encryptionEnabled = item.SSEConfiguration.SSE.enabled.toLower() == "true" ? true : false;
+                            // item.versionEnabled = item.VersioningConfiguration.Status.toLower() == "enabled" ? true : false;
```

Comment:
Please check if commenting this breaks anything in the List Bucket table.

Reply:
Because the back end of the previous code did not close, so the comment. I'll reupload it right away